### DISCUSSION TITLE
Quick search in favorites and routes

### DISCUSSION THREAD

**Discussion by DolomitesTom** [devemux86/cruiser#474] [Q&A] [answered]
Could you please enable an individual quick search in favorites and routes (both searches separated) which excludes POI and others? Just search filter with updating the filter in the saved routes or favorites after every character typing.

**Comment by devemux86** [comment, marked as answer]
@DolomitesTom thanks for the suggestion.

What do you mean with individual search?

Everything (favorites, routes, tracks, POI, locations) can be searched in the main search screen.
The results are grouped together, also with different icons.
And there is an option to select what to search in app settings.

**Comment by DolomitesTom** [comment]
Yes, the main search normaly includes POIs and this takes too long. When I am in the routes menu then I am only suffering for a quick search in my routes only to find them within a second.

**Reply by devemux86** [reply]
The **online** "Locations" search can usually take longer.
Can you disable the "Locations" in search settings and try again?

Quick search (or filter) inside the screen of favorites / routes / tracks is an interesting idea.
We could see how that might happen.
